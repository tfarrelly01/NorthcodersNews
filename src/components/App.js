import React from 'react';
// import PropTypes from 'prop-types';



/*
App.propTypes = {
  children: PropTypes.element.isRequired
};
*/

const App = () => (
  <div>
    <h3 className="title is-3">All Articles</h3>
    {/* this.props.children */}
  </div>
);

export default App;

