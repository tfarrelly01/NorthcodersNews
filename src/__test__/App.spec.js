import React from 'react';
// import renderer from 'react-test-renderer';
// used by enzyme
import { shallow } from 'enzyme';
// import { shallow, render, static } from 'enzyme';
import App from '../components/App';

test('App renders correctly', () => {
//  const component = renderer.create(<Search />);
//  const tree = component.toJSON();
//  expect(tree).toMatchSnapshot();

//  const component = shallow(<Search shows={preload.shows} />);

  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
