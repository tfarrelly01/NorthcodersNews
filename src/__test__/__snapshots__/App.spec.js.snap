// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <h3
    className="title is-3"
  >
    All Articles
  </h3>
</div>
`;
